packages/app-extensions/src/formField/formattedValueFactory.js

```diff
@@ -1,5 +1,21 @@
 import React from 'react'
 import {FormattedValue} from 'tocco-ui'
 
-export default type => (formField, modelField, formName, value, info, events, formData) =>
-  <FormattedValue type={type} value={value}/>
+const getOptions = (type, modelField, formData) => {
+  switch (type) {
+    case 'remote':
+    case 'multi-remote': {
+      return {
+        linkFactory: (key, children) =>
+          formData.linkFactory.detail(modelField.targetEntity, modelField.relationName, key, children)
+      }
+    }
+  }
+
+  return {}
```

Comment:
supposed to happen or worth to log a warning/error?

Reply:
thats ok